.vsts-ci.yml

```diff
@@ -291,6 +291,19 @@ extends:
               publishArgument: $(_publishArgument)
               officialBuildProperties: $(_officialBuildProperties)
               runTests: false
+      ### ARM64 TESTBUILD ###
+      - ${{ if or(eq(parameters.runTestBuild, true), eq(variables['Build.Reason'], 'PullRequest')) }}:
+        - template: /eng/pipelines/templates/jobs/sdk-job-matrix.yml@self
+          parameters:
+            pool:
+              name: Azure Pipelines
+              vmImage: macOS-latest
+              os: macOS
+            helixTargetQueue: osx.13.arm64
```

Comment:
`helixTargetQueue` is specified at the pipeline-level, because the pipelines run in different environments. The external (PR) pipeline uses Helix queues with `.open` on the end of the name. For the internal (Official) pipeline, we use the internal queues which **do not** have `.open` in the name.